sphinx/ext/apidoc.py

```diff
@@ -76,28 +77,27 @@ def is_packagedir(dirname: str | None = None, files: list[str] | None = None) ->
     return any(f for f in files if is_initpy(f))
 
 
-def write_file(name: str, text: str, opts: Any) -> None:
+def write_file(name: str, text: str, opts: CliOptions) -> Path:
```

Comment:
I hope this won't break anything (but I think only the 'quiet' option was not assumed to exist, right?)

Reply:
>  think only the 'quiet' option was not assumed to exist

why would we have to assume any options do not exist?
Am I missing something, `argparse` always populates every argument with a default no?

Reply:
Err.. yes it should. So I'm wondering why it was written like that. I thought at first that it was because we accepted whatever namespace but it's assuming that we are using argparse.

Reply:
Personally, I'm willing to take the risk, that all options are present 😅

Reply:
Me too. Because just after it's `opts.prefix` and it would raise here. Anyway, it someone is messing up with the CLI options, I'd say it's very unlikely that they would have all but the 'quiet' option.